test/smoke/src/helpers/setupEnvironmentHelper.ts

```diff
@@ -45,6 +45,49 @@ export class SetupEnvironmentHelper {
         SetupEnvironmentHelper.patchMetroConfig(workspacePath);
     }
 
+    public static  prepareHermesReactNativeApplication(workspaceFilePath: string, resourcesPath: string, workspacePath: string, appName: string, customEntryPointFolder: string, version?: string) {
+        let commandClean = path.join(workspacePath, "android", "gradlew") + " clean";
+        let commandBuild = path.join(workspacePath, "android", "gradlew") + " build";
+
+        console.log(`*** Executing  ${commandClean}...`);
+        cp.execSync(commandClean, { cwd: path.join(workspacePath, "android"), stdio: "inherit" });
+
+        let customEntryPointFile = path.join(resourcesPath, customEntryPointFolder, "App.js");
```

Comment:
I am not sure if we need independent `App.js` file for Hermes test as we can prepare all needed code for non Hermes application test and then use it in new test.

Reply:
Decided to remain separated `App.js` for Hermes smoke test.